src/typography/attributes.js

```diff
@@ -335,47 +330,14 @@ p5.prototype._updateTextMetrics = function() {
  * text('Have a wonderful day', 0, 10, 100);
  * </code>
  * </div>
+ *
  * <div>
  * <code>
  * textSize(20);
  * textWrap(CHAR);
  * text('Have a wonderful day', 0, 10, 100);
  * </code>
  * </div>
- * <div>
- * <code>
- * textSize(20);
- * textWrap(CHAR);
- * text('祝你有美好的一天', 0, 10, 100);
- * </code>
- * </div>
- * <div>
- * <code>
- * const scream = 'AAAAAAAAAAAAAAAAAAAAAAAAAAAAA';
- * textSize(20);
- * textWrap(WORD);
- * text(scream, 0, 0, 100);
- * fill(0, 0, 0, 75);
- * text(scream, 0, 20, 100);
- * fill(0, 0, 0, 50);
- * text(scream, 0, 40, 100);
- * fill(0, 0, 0, 25);
- * text(scream, 0, 60, 100);
- * strokeWeight(2);
- * ellipseMode(CENTER);
- * fill(255);
- * ellipse(15, 50, 15, 15);
- * fill(0);
- * ellipse(11, 47, 1, 1);
- * ellipse(19, 47, 1, 1);
- * ellipse(15, 52, 5, 5);
- * line(15, 60, 15, 70);
- * line(15, 65, 5, 55);
- * line(15, 65, 25, 55);
- * line(15, 70, 10, 80);
- * line(15, 70, 20, 80);
- * </code>
- * </div>
```

Comment:
The additional examples here are to demonstrate a couple more edge cases for `textWrap()` (ok maybe the last one a bit less but I selfishly like it). Do you think it would make sense to keep at least the Chinese characters one or it would just be confusing?

Reply:
The last example is a little busy. Good call to restore the Chinese characters. Fixed!